test/e2e/storage/vsphere/vsphere_volume_disksize.go

```diff
@@ -35,13 +34,12 @@ const (
 )
 
 /*
-	Test to verify disk size specified in PVC is being honored while volume creation.
+	Test to verify disk size specified in PVC is being rounded up correctly.
```

Comment:
It seems we don't run these tests at all, however, I updated this one to make sure it works.